examples/rust_bindings/src/rust_bindings_formatted.rs

```diff
@@ -1,118 +1,139 @@
 pub mod abigen_bindings {
     pub mod my_contract_mod {
-        use ::fuels::{
-            accounts::{Account, ViewOnlyAccount},
-            core::{
-                codec,
-                traits::{Parameterize, Tokenizable},
-                Configurables,
-            },
-            programs::{
-                contract::{self, ContractCallHandler},
-                logs::{self, LogDecoder},
-            },
-            types::{bech32::Bech32ContractId, errors::Result, AssetId},
-        };
-
-        pub struct MyContract<T: Account> {
-            contract_id: Bech32ContractId,
-            account: T,
-            log_decoder: LogDecoder,
+        #[derive(Debug, Clone)]
+        pub struct MyContract<A: ::fuels::accounts::Account> {
+            contract_id: ::fuels::types::bech32::Bech32ContractId,
+            account: A,
+            log_decoder: ::fuels::core::codec::LogDecoder,
+            encoder_config: ::fuels::core::codec::EncoderConfig,
         }
-        impl<T: Account> MyContract<T> {
+        impl<A: ::fuels::accounts::Account> MyContract<A> {
```

Comment:
What's the reason for specifying types this way (e.g., `::fuels::accounts::Account` instead of `Account`) all over this file?

Reply:
This file is an example of the expanded `abigen!` macro. We have not updated the file in some time and one of the changes is that we made the macro "hygienic". All types are used with unambiguous paths.